### PR TITLE
feat: Profile queries

### PR DESCRIPTION
**Pull request by jarulraj**
-- Added a basic timer
-- Need to check timing measurements

### CODE REVIEW

File: eva/server/command_handler.py [hunk 1]
```diff
@@ -24,20 +25,37 @@
 from eva.optimizer.statement_to_opr_convertor import StatementToPlanConvertor
 from eva.parser.parser import Parser
 from eva.utils.logging_manager import logger
+from eva.utils.timer import start_timer, end_timer, elapsed_time
 
 
-def execute_query(query) -> Iterator[Batch]:
+def execute_query(query, report_time: Boolean = False) -> Iterator[Batch]:
```
Comment (gaurav274) [inline eva/server/command_handler.py:31]: Boolean -> bool

File: eva/server/command_handler.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 import asyncio
 from typing import Iterator, Optional
+from xmlrpc.client import Boolean
```
Comment (gaurav274) [inline eva/server/command_handler.py:17]: Don't need this

File: eva/server/command_handler.py [hunk 3]
```diff
@@ -63,7 +81,8 @@ def handle_request(transport, request_message):
         logger.warn(e)
         response = Response(status=ResponseStatus.FAIL, batch=None, error=str(e))
     else:
-        response = Response(status=ResponseStatus.SUCCESS, batch=output_batch)
+        response = Response(status=ResponseStatus.SUCCESS,
```
Comment (gaurav274) [inline eva/server/command_handler.py:84]: Looks like you have a linter setting with a different max line length. Black uses 88

File: test/utils/test_timer.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+        self.assertTrue(t2 < 10)
+
+    def test_timer_with_query(self):
+
+        CatalogManager().reset()
+        create_sample_video(NUM_FRAMES)
+        load_query = """LOAD FILE 'dummy.avi' INTO MyVideo;"""
+        execute_query_fetch_all(load_query)
+        file_remove('dummy.avi')
```
Comment (gaurav274) [inline test/utils/test_timer.py:51]: There is nothing tested here. Can you verify if the elapsed time attribute is populated in the response and has a reasonable value? Thanks!

File: eva/server/command_handler.py (hunk 2 above)
Comment (jarulraj) [reply eva/server/command_handler.py]: Good catch. removed.

File: test/utils/test_timer.py (hunk 4 above)
Comment (jarulraj) [reply test/utils/test_timer.py:51]: Yes, added it in updated test cases.

**Pull request merged**